src/Overlay.tsx

```diff
@@ -68,8 +69,9 @@ export type OverlayProps = {
   onClickOutside: (e: TouchOrMouseEvent) => void
   onEscape: (e: KeyboardEvent) => void
   visibility?: 'visible' | 'hidden'
+  height?: keyof typeof heightMap
   [additionalKey: string]: unknown
-} & Omit<ComponentProps<typeof StyledOverlay>, 'visibility' | keyof SystemPositionProps>
+} & Omit<ComponentProps<typeof StyledOverlay>, 'height' | 'visibility' | keyof SystemPositionProps>
```

Comment:
I think these changes can all be removed as well

Reply:
Fixed in https://github.com/primer/components/commit/eb770c82b89c2d7659dd5f6fd06dfe50a80f368c